CONTRIBUTING.md

```diff
@@ -164,6 +165,48 @@ This repository contains several scripts and commands for performing numerous ta
 
 </details>
 
+## Experimental Cloudflare Deployment
+
+The website is in the process of being fully migrated and served from the [Cloudflare](https://www.cloudflare.com) network via [Cloudflare Workers](https://www.cloudflare.com/en-gb/developer-platform/products/workers/) using the [OpenNext Cloudflare adapter](https://opennext.js.org/cloudflare).
```

Comment:
I wouldn't mention this to be honest. Technically anyone can clone this repo and deploy on Vercel or Cloudflare, these instructions should be more of "how to deploy this on Cloudflare"